pages/conference/young-inspirers.vue

```diff
@@ -398,11 +339,30 @@ export default {
         }
     },
     methods: {
-        popupShow(index) {
-            this.popupIndex = index
+        showModal(item) {
+            this.isOpened = true
+            this.selectedItem = item
         },
-        popupClose() {
-            this.popupIndex = null
+        getSpeakerPhoto(speech) {
+            const photo = []
+            speech.speaker.forEach((element) => {
+                photo.push(element.photo)
+            })
+            return photo
```

Comment:
- 這幾個拿 speaker 資訊的 helper function 可以用 `.map()` 和 `.reduce()` 來取代 `.forEach()` 比較簡潔
- `data()` 裡面的 `speaker` 是一個 array，麻煩你幫忙改成 <code>speaker<b>s</b></code>
```suggestion
            return speech.speakers.map((speaker) => speaker.photo)
```

Reply:
沒用過 reduce，學到了😀